### PR TITLE
doc: release notes for v14.2.6

### PR DESCRIPTION
**Pull request by theanalyst**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/releases/nautilus.rst [hunk 1]
```diff
@@ -1,3 +1,16 @@
+v14.2.6 Nautilus
+================
+
+This is the sixth release of the Ceph Nautilus release series. This is a hotfix
+release primarily fixing a regression introduced in v14.2.5, all nautilus users are advised to upgrade to this release
```
Comment (neha-ojha) [inline doc/releases/nautilus.rst]:
```suggestion
release primarily fixing a regression introduced in v14.2.5, all nautilus users are advised to upgrade to this release.
```

**Comment by tchaikov** [comment]
```
/home/jenkins-build/build/workspace/ceph-pr-docs/doc/releases/general.rst:1: WARNING: Unknown target name: "14.2.6".
/home/jenkins-build/build/workspace/ceph-pr-docs/doc/releases/general.rst:1: WARNING: Unknown target name: "14.2.6".
```

File: doc/releases/nautilus.rst [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+
+Notable Changes
+---------------
+
+* This release fixes a `ceph-mgr bug that caused mgr becoming unresponsive on
```
Comment (neha-ojha) [inline doc/releases/nautilus.rst]:
```suggestion
* This release fixes a ``ceph-mgr`` bug that caused it to become unresponsive on
```

Comment (theanalyst) [reply doc/releases/nautilus.rst]: updated tnx

**Comment by kalebskeithley** [comment]
sixth update or seventh release.  Starting at 14.2.0.

**Comment by tchaikov** [comment]
jenkins render docs

**Pull request merged**